selection/select_ligand_and_surroundings/src/common/init.ts

```diff
@@ -0,0 +1,19 @@
+import { PluginContext } from "molstar/lib/mol-plugin/context";
+import { DefaultPluginSpec } from "molstar/lib/mol-plugin/spec";
+
+export async function createRootViewer() {
```

Comment:
we could probably just one have one common function for doing this for all examples

Reply:
We need each example folder to be self-contained.
It is repetitive, but it makes the code obvious to the reader.

We could instead have a script at the root of the repository to replicate the template in every example folder.